tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/RemotingTests.cs

```diff
@@ -0,0 +1,176 @@
+// <copyright file="RemotingTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETFRAMEWORK
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    [UsesVerify]
+    public class RemotingTests : TracingIntegrationTest
+    {
+        public RemotingTests(ITestOutputHelper output)
+            : base("Remoting", output)
+        {
+            SetServiceVersion("1.0.0");
+        }
+
+        public override Result ValidateIntegrationSpan(MockSpan span, string metadataSchemaVersion) => span.Name switch
+        {
+            "http.request" => span.IsWebRequest(metadataSchemaVersion),
```

Comment:
```suggestion
            "http.request" or "http.client.request" => span.IsWebRequest(metadataSchemaVersion),
```

Reply:
Whoopsies, I must have missed some of these 😅

Reply:
Fixed in a9f073d07f7f3d52b900299a6eb0d9d2a8e9a29b